recipes-security/optee/optee-fiovb_git.bb

```diff
@@ -27,4 +27,6 @@ EXTRA_OEMAKE = "OPTEE_CLIENT_EXPORT=${OPTEE_CLIENT_EXPORT} \
 do_install () {
     install -d ${D}${bindir}
     install -m 0755 ${S}/out/ca/fiovb ${D}${bindir}
+    ln -sf ${D}${bindir}fiovb ${D}${bindir}/fiovb_printenv
+    ln -sf ${D}${bindir}fiovb ${D}${bindir}/fiovb_setenv
```

Comment:
Better to do a relative link instead for both here, as they will all be under the same root folder.

Reply:
ok. maybe I will just do the same installation process that  other metas did (ie, meta-freescale...renaming the binary)